### PR TITLE
docs: add SECURITY.md

### PR DESCRIPTION
**Pull request by vankop**
> Which issue, if any, is this issue related to?

#4221 

> Is there anything in the PR that needs further explanation?

need to update emails list

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Reporting Security Issues
+
+If you discover a security issue in stylelint, please report it by sending an
+email to .
```
Comment (evilebottnawi) [inline SECURITY.md]: /cc @stylelint/core we need email here :smile:

Comment (jeddy3) [reply SECURITY.md]: Shall we just add `stylelint@gmail.com` for now to get this merged?

Comment (hudochenkov) [reply SECURITY.md]: Yeah. Is anyone checking this email? :)

Comment (jeddy3) [reply SECURITY.md]: I do :)

Comment (hudochenkov) [inline SECURITY.md]:
```suggestion
email to stylelint@gmail.com.
```

**Pull request merged**